src/likelihood_weighting.jl

```diff
@@ -1,21 +1,19 @@
 """
 p(fail) estimate across uniform space of the Gaussian process.
 """
-function p_estimate(gp, models; m=fill(500, length(models)), grid=true)
-    a₁, b₁ = models[1].range[1], models[1].range[end]
-    a₂, b₂ = models[2].range[1], models[2].range[end]
+function p_estimate(gp, models; num_steps=500, m=fill(num_steps, length(models)), grid=true)
```

Comment:
Is there any case where `m` is given manually to this function? 

I see its initialization depends on two arguments so I am wondering if we could demote this from function argument to simply function variable, initializing it at the start of the function.

Reply:
it's given [here](https://github.com/sisl/BayesianSafetyValidation.jl/blob/main/src/likelihood_weighting.jl#L30).  that said, there might be other places where we're doing something similar and never passing `m` explicitly.  but i can also imagine scenarios where you would want to discretize the different input dimensions differently, even if we're not necessarily doing so currently